safe/gui/tools/help/dock_help.py

```diff
@@ -91,20 +91,23 @@ def content():
         'There are three main areas to the dock panel:')))
     bullets = m.BulletedList()
     bullets.add(m.Text(
-        tr('the '),
-        m.ImportantText(tr(
-            'questions ')),
-        tr('area')))
+        # format 'the __questions__ area' for proper i18n
+        tr('the %s area') % (
+            m.ImportantText(tr(
+                'questions')).to_html(),
+        )))
```

Comment:
Hi @lucernae, I have tried way also, see my gitter chat last night. And it doesn't work properly since the ImportantText won't be rendered properly.